assimilation_code/programs/obs_sequence_tool/obs_sequence_tool.f90

```diff
@@ -79,6 +87,13 @@ program obs_sequence_tool
 integer :: copy_index_len      = 0
 integer :: qc_index_len        = 0
 
+
+! variables to manage whether or not the precomputed forward operator values
+! are written out
+
+integer :: FO_write_types(max_obs_input_types) = MISSING_I
+integer :: num_FO_types_to_write
```

Comment:
Since the existing behavior is to not write out any observation types ... I chose to let them choose which ones to write out (rather than which ones to suppress).